CRM/Core/EntityTokens.php

```diff
@@ -152,6 +152,7 @@ public function evaluateToken(TokenRow $row, $entity, $field, $prefetch = NULL)
    * @return bool
    */
   public function isHTMLTextField(string $fieldName): bool {
+    $metadata = $this->getMetadataForField($fieldName);
```

Comment:
Don't think we actually need this

Reply:
its in the 5.65 version but maybe not?

Reply:
it's a mistake - let's merge this & fix after

Reply:
I was doing a little follow up on event tokens and included removing this in there
https://github.com/civicrm/civicrm-core/pull/27349